### PR TITLE
Make instructor table fit two rows of links

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd & @Stanford-Online/openedx-courseops 

screen shot 2018-08-16 at 12 19 17 pm
(ignore the fact that the "graph this" is on the incorrect report)

### CODE REVIEW

File: lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
     .slickgrid {
         .slick-header-column.ui-state-default,
         .slick-row > .slick-cell {
-            padding: 5px;
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss:5]: Why don't we want the padding?

File: lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+            height: 100%;
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss:5]: Don't forget to port this to suclass :P

File: lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss (hunk 1 above)
Comment (caesar2164) [reply lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss:5]: Cause the text already has padding, and the padding takes up valuable real estate for the "graph this" link...

File: lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss (hunk 2 above)
Comment (caesar2164) [reply lagunita/lms/static/sass/partials/base/custom_overrides/_instructor.scss:5]: Haven't cut my version of suclass yet, no worries :)

**Review by stvstnfrd — APPROVED** [review]

**Comment by mondiaz** [comment]
Belated, but this looks good to me too! 👍

**Pull request merged**